### DISCUSSION TITLE
Image break in case it takes time to load from URL

### DISCUSSION THREAD

**Discussion by criticalAY** [community/community#69616] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

When adding an image in `a` tag and rendering in the README.md file the image (SVG) should load but it breaks if the `.svg` URL takes time to load for example this URL breaks in README.md file in github but works fine in local editors `https://opencollective.com/ankidroid/backers.svg?width=600` while this URL `https://opencollective.com/shields/backers.svg?width=600` works fine in the same

Screenshot
![image](https://github.com/community/community/assets/48384865/4bbac65f-419c-4c23-b024-2d9d5b27d1d8)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @criticalAY ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by criticalAY** [reply]
ok Thanks!
